### DISCUSSION TITLE
Custom Finder buttons that open folders like Applications/Downloads/Trash

### DISCUSSION THREAD

**Discussion by neilroy1998** [lawand-dot-io/taskbar#12] [Feature requests]
Quick access to opening finder windows for the Applications/Downloads/Trash/etc folders. As a bonus, support deleting files/folders dragged into the Trash button.

More context: https://github.com/lawand-dot-io/taskbar/discussions/262

**Comment by lawand-dot-io** [comment]
Interesting. Will consider this :)

**Reply by yo1995** [reply]
See uBar for an example - a trash bin shortcut and time display at the lower right corner would be helpful.

**Reply by lawand-dot-io** [reply]
@yo1995 thanks for the suggestion. will check out how uBar does it. and btw there's a separate feature request for a clock/cal: https://github.com/lawand-dot-io/taskbar/discussions/34

**Comment by Anthropos888** [comment]
The Trash icon with the possibility to delete items dragged on it would be very useful. Downloads / Applications as well. Additionally it would be great to have the possibility to add just any alias or file to the Taskbar for quick access.

**Comment by lawand-dot-io** [comment]
Duplicate request: https://github.com/lawand-dot-io/taskbar/discussions/69

**Comment by ShidoStrife** [comment]
+1 to this, shortcut to specific folder/file would be very useful.

**Comment by lawand-dot-io** [comment]
One user also requested an option to drag files into the trash icon to delete them. Writing this here to remember it later.
